### ISSUE TITLE
Support for images from clipboard?

### ISSUE THREAD

**Issue by orschiro** [SergKolo/indicator-bulletin#2]
Dear @SergKolo,

Would it be possible to support storing images from clipboard? :-)

Yours,

Robert

**Comment by SergKolo** [comment]
@orschiro  Hi, Robert ! Technically, yes , it is possible, but it might require having a temporary folder for storing those and I will need to play around with the code to leave a definitive answer on whether it can be implemented

What would you like the indicator to do with the images ? For instance, with the text there is a list of actions that one can perform with each item.  Do you have any suggestions on what should be done/which actions should be used with images ?

**Comment by orschiro** [comment]
Dear Serg, 

Thank you for your prompt response! 

I very appreciate it. 

I will think about possible actions to perform with images.

For now, I already consider pinning images a very useful feature as I am, for instance, a person who frequently reuses some image snippets to share in posts or emails.

Yours,

Robert

**Comment by albaphysic** [comment]
Yes, this feature would be very nice. Simply having images being listed and saved in clipboard history would be great.

**Comment by mlopezgva** [comment]
Hello. Actions?

- Pin
- Save (maybe _save as_ O:))
- Add image to Clipboard (= read from file and put it 'here')
There are actions like mirror/flip, reduce, convert., etc. (doable thru imagemagik or some other), but I don't think they are really, really important. Maybe Turn ±90º would be the most convenient or useful, aside pin and Save[ as].

Thank you for the tool, BTW, I was just looking for a clipboard manager with "pin" function. Great.

[Yes, I use a lot the "image in clipboard manager" feature in Clipman (XFCE)]
